index.d.ts

```diff
@@ -5,17 +5,20 @@
 
 export interface CsiMethod {
   src: string
-  dst?: string | undefined | null
-  operator?: boolean | undefined | null
+  dst?: string
+  operator?: boolean
 }
 export interface RewriterConfig {
   chainSourceMap?: boolean
   comments?: boolean
   localVarPrefix?: string
-  csiMethods?: Array<CsiMethod> | undefined | null
+  csiMethods?: Array<CsiMethod>
+}
+export interface ResultWithoutMetrics {
+  content: string
```

Comment:
And the `metrics`?

Reply:
NAPI interface is not used anymore so I skipped the metrics implementation